bin/flash.imx_usb

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
If you are downloading U-Boot over USB and "warm booting" it rather that writing it to flash and "cold booting" it, then you don't need any new flash.* implementation; simply use the existing flash.none (a/k/a flash_impl=none in the config file).